nowcasting_dataset/config/model.py

```diff
@@ -161,13 +161,13 @@ def check_start_and_end_datetime(cls, values):
 class PV(DataSourceMixin, StartEndDatetimeMixin):
     """PV configuration model"""
 
-    pv_filename: str = Field(
-        "gs://solar-pv-nowcasting-data/PV/PVOutput.org/UK_PV_timeseries_batch.nc",
-        description=("The NetCDF file holding the solar PV power timeseries."),
+    pv_filenames: List = Field(
+        ["gs://solar-pv-nowcasting-data/PV/PVOutput.org/UK_PV_timeseries_batch.nc"],
+        description=("List of the NetCDF file holding the solar PV power timeseries."),
```

Comment:
Tiny typo :slightly_smiling_face: 

```suggestion
        description=("List of the NetCDF files holding the solar PV power timeseries."),
```